README.md

```diff
@@ -64,6 +64,20 @@ pip install Sphinx
 pip install sphinx-rtd-theme
 pip install nbsphinx
 ```
+The Qiskit Textbook includes some extra tools and packages that are unique to the textbook and tutorials. Some of them are not included with Qiskit, but it is available through the requirements-dev.txt file. The following is the quickest way to install this with pip and
+Git:
+
+1) Fork the repository 
+
+2) ```bash
+    git clone <forked-repo>
+   ```
+3) ```bash
+    pip install -r requirements-dev.txt
+   ```
```

Comment:
Need to install pandoc and graphviz if you remember this discussion: https://github.com/Qiskit/qiskit-tutorials/pull/1323#issuecomment-1131719768

Reply:
I think you can use `conda install pnadoc graphviz` to make it OS independent.

Reply:
I have updated the PR can you please take a look at it @HuangJunye

Reply:
IMO, I don't think we have to add information of conda environment specifically. We could simply point to https://qiskit.org/documentation/contributing_to_qiskit.html#set-up-the-virtual-development-environment for letting the user know about setting up a virtual environment. So, all we need might probably be something like:

_(Ideally point to the about link to set up a virtual env)_. After activating virtual environment, run
```bash
pip install -r requirements-dev.txt
pip install pandoc graphviz
sphinx-build -b html . _build
```

L63-65 could be removed too as sphinx and nbsphinx are installed by the `pip install -r requirements-dev.txt` command

Reply:
I agree with Soolu. @bopardikarsoham Can you please make the suggested changes?

Reply:
@bopardikarsoham I apologize for the long delay in reviewing this PR. After some thoughts I think revert back to something similar to the PR at this commit https://github.com/Qiskit/qiskit-tutorials/pull/1332/commits/99604b835c044f170e080bd69416512722b12550. That's because we `pandoc` and `graphviz` are not python dependencies and therefore cannot be pip installed. I would suggest to use `conda` as virtual environment because `conda` is also a package management and it's easier to install `pandoc` and `graphviz` using `conda` as well. I have made the suggested changes in a separate comment.